editor/components/block-list/block.js

```diff
@@ -348,6 +354,96 @@ export class BlockListBlock extends Component {
 		this.setState( { error } );
 	}
 
+	onDragStart( event ) {
+		const dragInset = document.getElementById( `block-drag-inset-${ this.props.uid }` );
+		const block = document.getElementById( `block-${ this.props.uid }` );
+
+		// Closure to remove the cloned node from the DOM (fired within timeout below)
+		const removeBlockClone = ( _blockList, _cloneWrapper ) => {
+			return () => {
+				_blockList.removeChild( _cloneWrapper );
+			};
+		};
+
+		// Closure to hide the visible block and show inset in its place (fired within timeout below)
+		const showInset = () => {
+			return () => {
+				this.setState( state => Object.assign( {}, state, { dragging: true } ) );
+				document.body.classList.add( 'dragging' );
```

Comment:
I am not sure and I still need to work this out - it currently doesn't work as I like. Whatever I seem to try, the handler just changes randomly to "copy" in the middle of dragging sometimes.

Reply:
> Whatever I seem to try, the handler just changes randomly to "copy" in the middle of dragging sometimes.

Why does this happen? Also, what have you tried?

Reply:
Hey, I will focus on this as my next in line task. It's been sitting in my backlog for some time to be specific right now, but I will share details (or solution!) soon 😃

Reply:
Ok, should be fixed now - had to add a fake drag image to avoid browser defaults with dragging (Chrome changes the cursor apparently). Similar issue to what these folks seem to be having: https://github.com/RubaXa/Sortable/issues/246

I've added `dragging` to the body, cause we want the cursor change to be document wide. 

Any other concerns/suggestions on this?